ozaria/site/components/char_customization/PageCharCustomization/index.vue

```diff
@@ -0,0 +1,318 @@
+<script>
+  import Surface from '../common/Surface'
+  // TODO migrate api calls to the Vuex store.
+  import { getThangTypeOriginal } from '../../../../../app/core/api/thang-types'
+  import { mapGetters, mapActions } from 'vuex'
+
+  const { hslToHex } = require('core/utils')
+  const ThangType = require('models/ThangType')
+
+  const hslToHex_aux = ({ hue, saturation, lightness }) => hslToHex([hue, saturation, lightness])
+
+  const ozariaHeroes = {
```

Comment:
Will these come from database later?

Reply:
Added a comment. Unsure where to capture this as a task.
I think this would be good to put in a database.

Will add as eng sync topic.